.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask question
+    about: Ask a question about this project.
```

Comment:
"Ask a question about this project" reads like "I want to know what clap is about" instead of "I need help". Maybe rephrase?